_drafts/2023-10-09-first-time-things-to-bring.md

```diff
@@ -0,0 +1,42 @@
+
+## Notepad and pen/pencil, or a laptop and charger
+Perhaps you like your notes a bit more analog? A notepad is exactly what you need. Many people like to leave their laptops in their rooms entirely during talks and take notes on paper instead. Plus, it’s great for logging contact info of people you meet if you don’t have the time or patience to log their contact info into your phone’s contacts immediately.
+
+If you can't live without your laptop, make sure you bring your charging cable. There’s ample Wi-Fi at the conference provided by the Durham Convention Center, so you’ll have the internet at your disposal. Many attendees will have their laptops open to take notes, follow along with slides, and/or test out the concepts being presented during talks. Plus, it’s much easier to live-tweet talks from your computer than your phone if you’re into that sort of thing.
+
+## Refillable water bottle and/or coffee cup
+It’s pretty annoying getting thirsty in the middle of a conference talk. There will be access to all the water you can drink! Plus, you can be greener by reusing your own vessel instead of single-use disposable cups.
+
+There are coffee and tea in the Rio Vista Pavilion as well, so if you're a coffee/tea drinker, bring your own tumbler if you want extra for the talks.
+
+## Comfortable clothes
+Everyone is welcome, and dress casually and comfortably, whether that's shorts, pants, a dress, or whatever makes you feel good. It’s up to you to decide what you would like to wear. As long as it isn’t a Code of Conduct violation, it’s pretty much up to you.
+
+Some of the rooms can get a bit chilly, so you might want to have a jacket or sweater handy, but don't forget the weather outside!  It can get a little chilly in the evenings in Durham, NC.
+
+## Cell phone
+It’s practically essential for survival in suburban America. The mobile coverage is great in the venue area for all four major providers, so about any device will work.
+
+## Sunglasses and sunscreen
+It’s Durham, NC. It’s going to be a bit chillier than last year warm California weather. Preliminary forecasts show no meaningful chances of rain between the time I wrote this post and the tutorial day, with highs around 65-70 °F (18-21 °C) and lows around 45-48 °F (7-9 °C). There’s quite a bit of outdoor activities that we have suggested during your time here so don't forget that even if it is cold outside, the sun can still burn you!
+
+## Snacks
+The Convention Center will have full breakfast, lunch, and snacks during the breaks (11:00 AM and 3:40 PM on main talk days) on all conference days. If you’re an early riser like me, you’ll probably want a snack before breakfast (8:30 AM on Monday, 9:00 AM Tuesday and Wednesday). Also, if you have special dietary needs, it can’t hurt to have a spare snack on hand just in case.
```

Comment:
```suggestion
The Convention Center will have full breakfast, lunch, and snacks during the breaks (11:00 AM and 3:40 PM) on all conference days (Monday through Wednesday). If you’re an early riser like me, you’ll probably want a snack before breakfast (8:30 AM on Monday, 9:00 AM Tuesday and Wednesday). Also, if you have special dietary needs, it can’t hurt to have a spare snack on hand just in case.
```